src/models/user.py

```diff
@@ -1,3 +1,12 @@
 from remodel.models import Model
+from remodel.helpers import create_tables, create_indexes
+
 class User(Model):
```

Comment:
Ah I see, Alex and I may have misinterpreted that user is 'student' or 'instructor'. But the db schema says that the user belongs to either student or instructor, and student/instructor belongs to a section.